sdk/core/azure-core-serializer-json-gson/CHANGELOG.md

```diff
@@ -2,4 +2,4 @@
 
 ## 1.0.0-beta.1 (2020-05-04)
 
-- Initial release. Please see the README and wiki for information on the new design.
\ No newline at end of file
+- Initial release. Please see the README and wiki for information on the new design.
```

Comment:
I removed the extra changelog entry, that's already in master and just not in this branch yet. I don't know why this is still showing up as a change. For the moment, just let though.